Gemfile

```diff
@@ -1,5 +1,7 @@
 source "https://rubygems.org"
 
+ruby File.read('.ruby-version').strip if ENV["RACK_ENV"] == "production" # strict ruby version only on heroku
```

Comment:
That was the way that you had it originally 😉

I'd definitely down to put the ruby version in the Gemfile and remove the .ruby-version though .